### PR TITLE
Blog Post: LTS Grant for Jon Atack

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**
- [/blog/jon-atack-receives-lts-grant](https://os-website-git-lts-for-jon-atack-open-sats.vercel.app/blog/jon-atack-receives-lts-grant)

**Social Preview:**
- image

### CODE REVIEW

File: data/blog/jon-atack-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Jon is also receiving grants from [Maelstrom's Bitcoin Grant
+Program](https://maelstrom.fund/bitcoin-grant-program/), and [The Human Rights
+Foundation's Bitcoin Development Fund](https://hrf.org/devfund), allowing him to
+further focus on his vital work in Bitcoin development. Beyond the technical
+aspects, Jon hopes to pursue a long-term, evergreen role of help and service,
+with a measure of kindness, wisdom and forbearance.
+
+>My aim is to help improve Bitcoin for everyone, make it safer, easier to use,
+>and scale into the future.
```
[the remark this answers is not in the document]
Comment (Arvin21M) [reply data/blog/jon-atack-receives-lts-grant.mdx]: Looks good!

File: data/blog/jon-atack-receives-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+This grant allows Jon to resume full-time contributions, focusing on:
+
+- Conducting in-depth reviews of Bitcoin Core changes and BIP proposals to
+  safeguard security and robustness.
+
+- Mentoring developers and guiding complex changes in Bitcoin's codebase.
+
+Jon is also receiving grants from [Maelstrom's Bitcoin Grant
+Program](https://maelstrom.fund/bitcoin-grant-program/), and [The Human Rights
+Foundation's Bitcoin Development Fund](https://hrf.org/devfund), allowing him to
```
[the remark this answers is not in the document]
Comment (Arvin21M) [reply data/blog/jon-atack-receives-lts-grant.mdx]: Updated

**Review by dergigi — APPROVED** [review]
Looks good to me; scheduled to go live today ⏳

**Comment by dergigi** [comment]
Live now!
- https://opensats.org/blog/jon-atack-receives-lts-grant
- [nostr post](https://njump.me/nevent1qqsydwxap0rtta74nrnzuykqpflxswng7p2zrxxql9zwuql4c8986ugpzemhxue69uhhyetvv9ujuurjd9kkzmpwdejhgqg5waehxw309aex2mrp0yhxgctdw4eju6t0qgs8suecw4luyht9ekff89x4uacneapk8r5dyk0gmn6uwwurf6u9ruscln3rd)

**Pull request merged**